completions/fish/crio.fish

```diff
@@ -42,6 +42,7 @@ complete -c crio -n '__fish_crio_no_subcommand' -f -l default-sysctls -r -d 'Sys
 complete -c crio -n '__fish_crio_no_subcommand' -f -l default-transport -r -d 'A prefix to prepend to image names that cannot be pulled as-is'
 complete -c crio -n '__fish_crio_no_subcommand' -f -l default-ulimits -r -d 'Ulimits to apply to containers by default (name=soft:hard) (default: [])'
 complete -c crio -n '__fish_crio_no_subcommand' -f -l enable-metrics -d 'Enable metrics endpoint for the server on localhost:9090'
+complete -c crio -n '__fish_crio_no_subcommand' -f -l enable-custom-shm-size -d 'Enable users to set custom shm size instead of default value 64M'
```

Comment:
```suggestion
complete -c crio -n '__fish_crio_no_subcommand' -f -l enable-custom-shm-size -d 'Enable users to set custom shm size instead of using the default value 64M'
```